modules/affile/logproto-file-writer.c

```diff
@@ -32,6 +33,9 @@
 typedef struct _LogProtoFileWriter
 {
   LogProtoClient super;
+  SignalSlotConnector *signal_slot_connector;
```

Comment:
this should probably named to resemble the functionality that is associated with the slot, e.g. file_rotation_signal or something like that.

Reply:
I've updated the name